modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1787,14 +1789,18 @@ private CompletableFuture<Void> handleChangePendingAssignmentEvent(
                                     stringKey, partId, table.name(), localNode().address(), pendingAssignments, revision);
                         }
 
-                        return setTablesPartitionCountersForRebalance(replicaGrpId, revision, pendingAssignments.force())
+                        // TODO: should come from the assignments. The version valid at the time of assignment creation.
```

Comment:
Better to put https://issues.apache.org/jira/browse/IGNITE-22661 here, or just check that your actually did update this in https://issues.apache.org/jira/browse/IGNITE-22661 iself.